README.md

```diff
@@ -1,37 +1,134 @@
-## Prerequisites
-* [We used Jammy ubuntu VM ](https://github.com/Alliedium/awesome-proxmox)
-* [Ansible is installed](https://github.com/Alliedium/awesome-ansible/)
+## This ansible playbook installs the [`jenkins`](https://www.jenkins.io/doc/) on the specified host ##
+
+# Prerequisites
+* Run the ansible playbook on `Debian` or `Ubuntu` Linux distribution. [We used Jammy ubuntu VM ](https://github.com/Alliedium/awesome-proxmox). Use the [script](https://github.com/Alliedium/awesome-proxmox/blob/main/vm-cloud-init-shell/.env.example) to create VM on `Proxmox`.  
```

Comment:
Remove "Linux distribution", We used Jammy ubuntu VM -> Used was VM with Jammy Ubuntu